src/llmcompressor/args/training_arguments.py

```diff
@@ -30,3 +30,7 @@ class TrainingArguments(HFTrainingArgs):
             "checkpoints will be written."
         },
     )
+
+    @property
+    def place_model_on_device(self):
```

Comment:
a user would never want to set this to `True`? wondering if it's better off as an optional input that defaults to False

Reply:
> We want this argument to be False as we handle the device map and model initialization based on the calibration needs

nm, LGTM

Reply:
No we dont want them controlling this as of yet. 

We provide options to update the device map and change how the model is mapped when running oneshot and in general for llmcompressor.

We then pass that to the Trainer so we're essentially already doing the device map heavy lifting.

I think the downside is that we're exposing this property at all which can be confusing for somebody looking through the args, but this is not super urgent to fixc as we're not yet focused on improving training workflows.